batch/file-ingest/src/main/java/org/springframework/ingest/mapper/fieldset/PersonFieldSetMapper.java

```diff
@@ -0,0 +1,15 @@
+package org.springframework.ingest.mapper.fieldset;
+
+import org.springframework.ingest.domain.Person;
+import org.springframework.batch.item.file.mapping.FieldSetMapper;
+import org.springframework.batch.item.file.transform.FieldSet;
+
+public class PersonFieldSetMapper implements FieldSetMapper<Person> {
```

Comment:
Given previous comments, I wouldn't expect this to need to remain.

Reply:
per response; i would prefer making things immutable where possible but if this adds distraction to the sample i can remove it. let me know which you would prefer